scripts/install-requirements.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Detect the operating system type
+OS="$(uname -s)"
+DISTRO=""
+
+# Define installation functions
+install_requirements_ubuntu() {
+    echo "Installing requirements and dependencies on Ubuntu..."
+    sudo apt-get update
+    sudo apt-get install -y libleveldb-dev sqlite3 libsqlite3-dev libunwind8-dev
+}
+
+install_requirements_fedora() {
+    echo "Installing requirements and dependencies on Fedora..."
+    sudo dnf install -y leveldb-devel sqlite sqlite-devel libunwind-devel
+}
+
+install_requirements_opensuse() {
+    echo "Installing requirements and dependencies on openSUSE..."
+    sudo zypper install -y libleveldb-devel sqlite3 libsqlite3-devel libunwind-devel
```

Comment:
Don't forget to add refresh **no** `-y`
```sh
zypper refresh
```

Also correct command is
```sh
zypper install -y leveldb-devel sqlite3-devel
```

Complete command is:
```sh
zypper refresh && zypper install -y leveldb-devel sqlite3-devel
```